meta/runtime.yml

```diff
@@ -187,4 +187,4 @@ plugin_routing:
       redirect: theforeman.foreman.sync_plan
     katello_upload:
       redirect: theforeman.foreman.content_upload
-requires_ansible: '>=2.9'
+requires_ansible: '>=2.9.17'
```

Comment:
technically, support for collections is there since 2.9.10, but collection roles are somewhat broken until .17 and this seemed like the safer boundary.